Localization/Translations/DebugMessages.lua

```diff
@@ -146,7 +146,7 @@ local debugMessagesLocales = {
         ["esES"] = "Se produjo un error al obtener los objetivos para %s %s %s %s",
         ["frFR"] = "Une erreur s'est produite lors de la création des objectifs pour %s %s %s %s",
     },
-    ["Error: Questie tracker in invalid location, resetting..."] = {
```

Comment:
There are two usages of this string, which also need to be changed. Otherwise no translation will be found.

Both are in "TrackerBaseFrame.lua"